apps/frontend/src/pages/projects/projects.tsx

```diff
@@ -50,6 +51,8 @@ const Projects = (): JSX.Element => {
 		<>
 			<Header />
 			<p>Current path: {pathname}</p>
+			{/*  Breadcrumbs test here */}
```

Comment:
```suggestion
```

Reply:
Sorry, that was just for me while making the test, this will be removed.